chat-plugins/chat-monitor.js

```diff
@@ -100,7 +88,7 @@ let chatfilter = function (message, user, room) {
 		let [line, reason] = filterWords.autolock[i];
 		let matched = false;
 		if (typeof line !== 'string') continue; // Failsafe to appease typescript.
-		if (line.endsWith('\\b')) {
+		if (line.startsWith('\\b') || line.endsWith('\\b')) {
 			matched = (new RegExp(line, 'g')).test(lcMessage);
```

Comment:
You don't need `'g'` for autolock or warn filters. It either matched or it didn't.

Reply:
Also you don't need parens around `new RegExp()`